### ISSUE TITLE
What is the meaning of "RandomIDCount" in Exposure Checks data?

### ISSUE THREAD

**Issue by vfilimonov** [DP-3T/dp3t-app-ios-ch#264]
Apple allows to export exposure checks data (Settings -> Privacy -> Health -> COVID-19 Exposure Logging -> Exposure Checks -> Export Exposure Checks). I believe this is exactly the data collected and used by "SwissCovid" app (at least each record there is tagged by `ch.admin.bag.dp3t` as DataSource).

I'm interested what does `RandomIDCount` field stands for? Intuitively I it would mean the number of encounters. But records look quite confusing. For example my today's log:
```
Timestamp                  RandomIDCount
2020-08-18 06:01:21+02:00            131
2020-08-18 06:01:21+02:00            140
2020-08-18 06:01:21+02:00            136
2020-08-18 06:01:21+02:00            127
2020-08-18 06:01:21+02:00             98
2020-08-18 06:01:21+02:00             69
2020-08-18 06:01:21+02:00              6
2020-08-18 06:01:21+02:00             48
2020-08-18 06:01:21+02:00             28
```

I.e. it records several hundreds encounters while I was still sleeping. How would one interpret this?

Furthermore, I have about 1500 encounters on days when I stay at home and don't leave apartments. And what is even stranger - the number of encounters per day (1500-1600) does not depend on whether I stay at home full day, or spend an hour in Coop/Migros, or stay in bar/restaurant, or use public transport. 

This is for example the total number of RandomIDCount of my last 5 days which included all of the above mentioned cases (from no real contacts per day to hundreds of real contacts per day):
```
2020-08-13    1457  (most active day -> a lot of real encounters)
2020-08-14    1570
2020-08-15    1642  (stay-at-home day)
2020-08-16    1616  (stay-at-home day)
2020-08-17    1538
```

Could you please clarify this question? Since this is the data collected and used by app it raises some suspicion about reliability of the system.

**Comment by UBaggeler** [comment]
The "Export Exposure Checks" button exports the same information that is shown above this button, but in a machine readable format (JSON). When looking at one specific exposure check in the settings app `RandomIDCount` is translated to `Provided Key Count`. This is the number of `TEK`s ([Temporary Exposure Key](https://www.apple.com/covid19/contacttracing/)) which SwissCovid downloaded from the CDN for a certain day and passed onto the ExposureNotification API of Apple. `Matched Key Count` is the number of encounters which matched with these `TEK`s. Having a match does not necessarily mean that you are shown a warning in SwissCovid, as it only says that your phone once registered an ID which later tested positive. SwissCovid only shows a warning if you were long enough and close enough to this other person.

To summarize: The `RandomIDCount`/`ProvidedKeyCount` has nothing to do with how many encounters were registered (this is never revealed by iOS/Android) but shows how many keys of positively tested persons for a certain day were downloaded and checked on your phone.

**Comment by vfilimonov** [comment]
Great, thanks a lot for the explanations, @UBaggeler 
I believe you could close the issue
